code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -351,7 +351,7 @@
 						UpdateName()
 						to_chat(user, "Name changed to [new_name].")
 
-					if("Appearance")
+					if("Appearance") //what a snowflake fuck you
```

Comment:
eh, I might in a future pr
i'll remove the comment because I want this merged for now